src/dtgtk/paint.c

```diff
@@ -1546,7 +1546,7 @@ void dtgtk_cairo_paint_label_sel(cairo_t *cr, gint x, gint y, gint w, gint h, gi
   PREAMBLE(1, 0, 0, 0)
 
   const double r = 0.4;
-
+  const double lw = 0.15;
```

Comment:
Using 0.1 here I find it more readable:

![image](https://user-images.githubusercontent.com/467069/157202286-bd52dad1-c66c-48ba-8655-f3dff2415927.png)

Reply:
Otherwise for me it is better this way.

Reply:
@TurboGit I agree your screenshot looks more betterer